media-gfx/netgen/netgen-6.2.2204.ebuild

```diff
@@ -0,0 +1,155 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_10 )
+inherit cmake desktop python-single-r1 xdg
+
+DESCRIPTION="Automatic 3d tetrahedral mesh generator"
+HOMEPAGE="https://ngsolve.org/ https://github.com/NGSolve/netgen"
+SRC_URI="https://github.com/NGSolve/netgen/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="LGPL-2.1"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="ffmpeg gui jpeg logging mpi opencascade python test"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	ffmpeg? ( gui )
+	jpeg? ( gui )
+	python? ( gui )
+"
+
+DEPEND="
+	sys-libs/zlib
+	ffmpeg? ( media-video/ffmpeg:= )
+	gui? (
+		dev-lang/tcl:0/8.6
+		dev-lang/tk:0/8.6
+		media-libs/glu
+		virtual/opengl
+		x11-libs/libX11
+		x11-libs/libXmu
+		x11-libs/libxcb:=
+	)
+	jpeg? ( media-libs/libjpeg-turbo:0= )
+	logging? ( dev-libs/spdlog:= )
+	mpi? (
+		sci-libs/metis
+		virtual/mpi
+	)
+	opencascade? ( sci-libs/opencascade:= )
+	python? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep '
+			dev-python/pybind11[${PYTHON_USEDEP}]
+			'
+		)
+		mpi? (
+			$(python_gen_cond_dep 'dev-python/mpi4py[${PYTHON_USEDEP}]' )
+		)
+	)
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	sys-apps/lsb-release
+	virtual/imagemagick-tools[png]
```

Comment:
This dependency we can we can put behind a `gui? ( ... )`. Since we only call the convert command if we build the gui.

Reply:
good catch, I overlooked that